### DISCUSSION TITLE
Toast notification upon predicting

### DISCUSSION THREAD

**Discussion by SylvainChevalier** [Metaculus/metaculus#899] [Ideas]
In the old site, predicting triggered a toast popup at the top of the page. It was a nice piece of visual feedback that your prediction went through, and it made it feel less like shouting in the void.
@aseckin @hickman-santini what do you think? We could do some more heavier "celebration"-type thing.

**Comment by hickman-santini** [comment]
I'm always pro celebration - confetti and/or rockets animation. But a "Thanks for forecasting!" toast or something would also be nice.
